authz/rbac_translator_test.go

```diff
@@ -311,3 +748,15 @@ func TestTranslatePolicy(t *testing.T) {
 		})
 	}
 }
+
+func anyPbHelper(in map[string]interface{}) *anypb.Any {
+	pb, err := structpb.NewStruct(in)
+	if err != nil {
+		panic(err)
```

Comment:
panic is generally discouraged. You could pass *testing.T here and do `t.Fatal(err)`

Reply:
Changed